### PR TITLE
Update contracts.json

### PR DESCRIPTION
**Pull request by iSkitzTV**
Add CatWifHat and Catnip Tokens from the CatWifHat Ecosystem Project

### CODE REVIEW

**Review by StrathCole — APPROVED** [review]

File: columbus-5/cw20/contracts.json [hunk 1]
```diff
@@ -1377,6 +1377,18 @@
...
+        "symbol": "CatWifHat",
+        "decimals": 6,
+        "logo": "https://i.ibb.co/Drv3V8p/Lion-Cat-5.png",
+        },
+        "terra10quf590wyr8tuf3pp05hgwm94n3dgfnma377uwaynl3tj47df7nskwdxgs": {
+        "name": "CatWifHat Ecosystem Token",
+        "symbol": "Catnip",
+        "decimals": 6,
+        "logo": "https://i.ibb.co/Cz2mHQ4/catnip-token.png",
+        },
```
Comment (StrathCole) [inline columbus-5/cw20/contracts.json]: No trailing comma in the last entry allowed.

**Review by StrathCole — CHANGES_REQUESTED** [review]
See comment

File: columbus-5/cw20/contracts.json (hunk 1 above)
Comment (StrathCole) [inline columbus-5/cw20/contracts.json]:
```suggestion
        }
```

**Comment by iSkitzTV** [comment]
Thank you dude, Was just about to do this, think I was too tired last night! :D

**Pull request merged**